Source/Assimil8or/Assimil8or.cpp

```diff
@@ -667,375 +894,21 @@ void Assimil8orPreset::parse (juce::StringArray presetLines)
         }
 
         LogParsing (juce::String (scopeDepth) + "-" + presetLine.trimStart ());
-        const auto key { presetLine.upToFirstOccurrenceOf (":", false, false).trim () };
-        const auto value { presetLine.fromFirstOccurrenceOf (":", false, false).trim () };
-
-        auto keyIs = [&key] (const juce::String& desiredKey)
+        key = presetLine.upToFirstOccurrenceOf (":", false, false).trim ();
+        key = key.upToFirstOccurrenceOf(" ", false, false);
+        value = presetLine.fromFirstOccurrenceOf (":", false, false).trim ();
+        auto keyIs = [this] (const juce::String& desiredKey)
         {
             return key.upToFirstOccurrenceOf (" ", false, false) == desiredKey;
         };
-        auto getParameterIndex = [&key] ()
+        if (!keyIs(""))
```

Comment:
I think we can remove this check, as it will be caught in `dispatch()`. oh, actually, you can move the body of dispatch here, since it is not called anywhere else, and the body of `parse` has shrunk dramatically

Reply:
Moved it to a lambda!